install.sh

```diff
@@ -190,8 +191,12 @@ download() {
   curl --proto '=https' --tlsv1.2 --location --silent --fail --show-error --output "$install_path/bin/ockam" "$_url"
   info "Downloaded ockam binary at the specified directory: $install_path/bin/ockam"
 
-  info "Granting permission to execute: chmod u+x $install_path/bin/ockam"
+  info "Downloading scripts"
+  curl --proto '=https' --tlsv1.2 --location --silent --show-error --output "$install_path/install.sh" "$_script_base_url/install.sh"
+
+  info "Granting permission to execute: chmod u+x $install_path/bin/ockam, $install_path/install.sh"
```

Comment:
Instead of saving the old installer, I think it would be safer to always install a new version using the current install.sh in our git repo. This way we can tackle any breaking changes in the installers asumptions.

Reply:
Yeah I like that idea! That'll save needing to have the installer local on the users computer too. I'll make those changes.